CHANGELOG.md

```diff
@@ -15,6 +15,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add missing data source to atlas dashboards.
 - Fix missing provider specific dashboards.
+- Update alertmanager-overview dashboard (angular deprecation).
+- Update fluentbit dashboard (angular deprecation).
+- Update operatorkit dashboard (angular deprecation).
```

Comment:
Adding changelog for https://github.com/giantswarm/dashboards/pull/558/files and https://github.com/giantswarm/dashboards/pull/557 as well